src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java

```diff
@@ -264,7 +274,7 @@ private XmlFile getConfigFile() {
     @Extension
     public static class ProvisioningListener extends CloudProvisioningListener {
 
-        private final CloudStatistics stats = CloudStatistics.get();
+        private final CloudStatistics stats = getCloudStatistics();
```

Comment:
> That not a preference, that how human friendly and readable code should look like.

Of course it is, you just refuse to admit other people do not share your religion :smile:.